src/shogun/kernel/GaussianKernel.h

```diff
@@ -114,18 +114,6 @@ class CGaussianKernel: public CDotKernel
 		 */
 		virtual float64_t get_width() const;
 
-		/** set the compact option
```

Comment:
The header file also had the compact mentioned in its class description. Please update as well